olm/docker-run.sh

```diff
@@ -43,9 +43,9 @@ check() {
 build() {
   printf "%bBuilding image %b${IMAGE_NAME}${NC}..." "${BOLD}" "${BLUE}"
   if docker build --build-arg OPERATOR_SDK_VERSION=${OPERATOR_SDK_VERSION} -t ${IMAGE_NAME} > docker-build-log 2>&1 -<<EOF
-  FROM evns/yq
+  FROM golang:1.14-alpine
```

Comment:
Should we use `golang:1.13-alphine` since we switched from 1.12 to 1.13 version ?